modules/analysis-common/build.gradle

```diff
@@ -25,12 +25,12 @@ dependencies {
   compileOnly project(':modules:lang-painless')
 }
 
-tasks.named("yamlRestTestV7CompatTest").configure {
-  systemProperty 'tests.rest.blacklist', [
-       //marked as not needing compatible api
-       'indices.analyze/10_analyze/htmlStrip_deprecated', // Cleanup versioned deprecations in analysis #41560
-       'analysis-common/40_token_filters/delimited_payload_filter_error', //Remove preconfigured delimited_payload_filter #43686
-       'analysis-common/20_analyzers/standard_html_strip', // Cleanup versioned deprecations in analysis #41560
-       'search.query/50_queries_with_synonyms/Test common terms query with stacked tokens', // #42654 -  `common` query throws an exception
-    ].join(',')
-}
+
+tasks.named("yamlRestTestV7CompatTransform").configure(
```

Comment:
FWIW, you shouldn't need the parens here. This can just be `configure {` as it was before. I'd do the same in the other build scripts as well to reduce verbosity.

Reply:
done. thanks.